source/usage-examples/bulkWrite.txt

```diff
@@ -0,0 +1,86 @@
+=======================
+Perform Bulk Operations
+=======================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+MongoDB provides clients the ability to perform write operations in bulk.
+Bulk operations affect a *single* collection.
+
+The following code sample performs a bulk write operation against the
+``users`` collection in the ``sample_mflix`` database. Specifically, the
+program uses the :node-api:`collection.bulkWrite
+<Collection.html#bulkWrite>` method and ``insertOne`` operations to write
+the user information from a ``users.json`` file to the ``users``
+collection. Download the dataset here: `users.json <https://raw.githubusercontent.com/mongodb-university/universal-driver-examples/master/users.json>`_
+
+``bulkWrite()`` accepts the following parameters:
+
+- ``operations`` (``Array.<object>``): specifies the bulk operations to
+  perform. Each operation is passed to ``bulkWrite()`` as an object in
+  an array.
+
+- ``options`` (``object``): *optional* settings that affect the execution
+  of the operation, such as :manual:`write concern
+  </reference/write-concern>` and order.
+
+  By default, MongoDB executes bulk operations in the specified order
+  (i.e. serially). When executing operations in order, if an error occurs
+  during the processing MongoDB returns without processing any of the
+  remaining operations in the list. In contrast, when ``ordered`` is
+  ``false``, MongoDB continues to process remaining write operations in
+  the list. Unordered operations are faster as MongoDB can execute the
+  operations in parallel, but the results of the operation may vary. For
+  example, a ``deleteOne`` operation run before an ``updateMany``
+  operation might have a different result from running it after the
+  ``updateMany``. Refer to :ref:`Execution of Operations
+  <bulkwrite-write-operations-executionofoperations>` for more
+  information.
```

Comment:
Big fan of this example here, I think you nailed our approach of "explain a concept in an approachable way, without too much jargon, and link to some reference doc". Perfect.